snuba/utils/streams/batching.py

```diff
@@ -330,29 +301,28 @@ def _commit(self) -> None:
                     raise
```

Comment:
I guess a lot of this should go in `KafkaConsumer` now. I don't even really remember why we did a lot of this. 🤷‍♂

Reply:
I moved this, I don't think it's necessarily a good solution to whatever problem we were running into here (it's probably not a big deal if we can't commit, unless we are trying to commit as part of rebalancing — then it is a big deal) but it works well enough for now.